### PR TITLE
added jpeg-support:

### PR DESCRIPTION
**Pull request by mirzazeyrek**
example images in jpg format.
jpg example in example.js
added jpeg-js in resemble.js
added extension check in LoadImageData method.

### CODE REVIEW

**Comment by lksv** [comment]
Thanks you for commit. Cloud you please:
* use 2 spaces for nested block (not tap as in resemble.js, not 4 spaces as in example.js).
* please do not upgrade version in package.json (I will do it later in separate commit) and remove your extra space at the end of "dependences" line.

**Comment by mirzazeyrek** [comment]
@lksv You are welcome lukas. I have made the changes and I hope I didn't missed anything :)

File: resemble.js [hunk 1]
```diff
@@ -124,13 +124,13 @@ var _this = {};
 				console.log(ext);
```
Comment (lksv) [inline resemble.js]: please remove the `console.log`

**Comment by mirzazeyrek** [comment]
@lksv console.log is removed.

**Comment by lksv** [comment]
sorry that I didn't notice `console.log` first time. I think there are still tabs in resemble.js file. Could you please squash to commits to single one.

... or maybe I can substitute tabs while merging (I can do it at eventing/night today).

**Comment by lksv** [comment]
It's a pity that you closed it. I would very like to merge it. Can I?

**Comment by mirzazeyrek** [comment]
@lksv I am removing previous commits and trying to merge everything in one commit :)

**Comment by mirzazeyrek** [comment]
@lksv It looks like as you requested now. Of course you can merge it.

**Comment by lksv** [comment]
@mirzazeyrek I cannot merge it when it is closed.

**Comment by mirzazeyrek** [comment]
@lksv yes could be something wrong with the git. I reset the branch and made the changes once again. Please check if everything looks ok.

**Pull request merged**